src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -492,3 +495,41 @@ def get_data_cfg(cfg, subset):
                 dataset = dataset.dataset
             return dataset
         return cfg.data[subset]
+
+    @staticmethod
+    def adapt_input_size_to_dataset(
```

Comment:
IMHO, it is more suitable to move this function in `InputSizeManager`. How do you think about?

Reply:
Good point, but I didn't want to let `InputSizeManager` know about `DatasetEntity` and others.
Maybe, it will directly handle `dm.Dataset` in the near future.